scripts/prerelease/generate_prerelease_overlay_script.py

```diff
@@ -57,9 +57,13 @@ def main(argv=sys.argv[1:]):
              "'catkin_workspace/src')"
     )
     parser.add_argument(
-        '--json', action='store_true',
+        '--json', action='store_true', default=False,
```

Comment:
There is no need to add `default=False` here (or on the new option). The default default value is `None` which is sufficient.